src/rt-app_types.h

```diff
@@ -42,9 +42,13 @@ Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 #define EXIT_INV_CONFIG 2
 #define EXIT_INV_COMMANDLINE 3
 
+/* It is not available in headers without defining __USE_GNU macro */
+#define SCHED_IDLE 5
```

Comment:
can you add a #ifndef __USE_GNU to prevent multiple definition ?

Reply:
Okay,.

Reply:
Vincent, I have made the change you were looking for. Thanks.

Reply:
Could you rephrase to something like "SCHED_IDLE is not available if __USE_GNU is not defined" ?
Also, I'd update changelog, since we now have a reasonable solution to this problem. :-)